resolwe_bio/tests/files/ALL_SE_sorted1.bed

```diff
@@ -0,0 +1,1859 @@
+chr1	9996	40602	1_group4_wnt_1559_lociStitched
```

Comment:
Would it make sense to shorten this test bed file? Does it have any influence on the time needed for the tests to run?

Also, existing files in `resolwe_bio/tests/files` usually use lower case for names and the prefix says something where this is used. Could you improve the name?